examples/deploy_service/README.md

```diff
@@ -16,6 +16,7 @@ It will:
 |------|-------------|:----:|:-----:|:-----:|
 | cluster\_name\_suffix | A suffix to append to the default cluster name | string | `""` | no |
 | compute\_engine\_service\_account | Service account to associate to the nodes in the cluster | string | n/a | yes |
+| enable\_binary\_authorization | Enable BinAuthZ Admission controller | string | `"false"` | no |
```

Comment:
These tables should be edited only through `make generate_docs`, not manually; this variable has not been added to any of the examples.

```suggestion
```